crates/typst/src/doc.rs

```diff
@@ -55,12 +55,7 @@ impl Frame {
     #[track_caller]
     pub fn new(size: Size, kind: FrameKind) -> Self {
         assert!(size.is_finite());
-        Self {
-            size,
-            baseline: None,
-            items: Arc::new(vec![]),
-            kind,
-        }
+        Self { size, baseline: None, items: <_>::default(), kind }
```

Comment:
I believe it's fine to explicitly indicate the default as being an empty vector, although I understand the change. I believe keeping `Arc::new` is still relevant / a more important principle though.

Reply:
Agreed. `<_>::default()` is ugly and less clear